db/dbfunctions.py

```diff
@@ -1347,4 +1347,8 @@ def get_rawfiles_for_taskentry(e, system_subfolders=None):
         else:
             files[df.system.name] = filepath
 
+    params = e.task_params
+    if 'neuropixel_port1_site' in params:
```

Comment:
in old data did you always have 'neuropixel_port1_site' or was it something else? maybe you could just check if 'neuropixel' is in e.features?